kafka/src/test/java/org/axonframework/extensions/kafka/eventhandling/consumer/FetchEventsTaskTest.java

```diff
@@ -135,7 +135,7 @@ void testFetchEventsTaskInterruptionClosesAsExpected() {
 
 
         assertWithin(Duration.ofMillis(TIMEOUT_MILLIS), () -> assertTrue(expectedToBeClosed.get()));
-        verify(testConsumer).close();
+        verify(testConsumer, timeout(TIMEOUT_MILLIS)).close();
```

Comment:
we could make `TIMEOUT_MILLIS` based on the KafkaConsumer's `DEFAULT_CLOSE_TIMEOUT_MS`

Reply:
As long as the used value is consistent within the test, I'm fine with it.